cocos/core/gfx/base/define.ts

```diff
@@ -103,20 +103,6 @@ export enum SurfaceTransform {
 }
 
 export enum Feature {
-    COLOR_FLOAT,
-    COLOR_HALF_FLOAT,
-    TEXTURE_FLOAT,
-    TEXTURE_HALF_FLOAT,
-    TEXTURE_FLOAT_LINEAR,
-    TEXTURE_HALF_FLOAT_LINEAR,
-    FORMAT_R11G11B10F,
-    FORMAT_SRGB,
-    FORMAT_ETC1,
-    FORMAT_ETC2,
-    FORMAT_DXT,
-    FORMAT_PVRTC,
-    FORMAT_ASTC,
-    FORMAT_RGB8,
```

Comment:
This looks like breaking changes, if user have used these Feature flags in previous version, what will happen in the new version ?

Reply:
Yes, we can add some deprecations for this